client/src/app/pages/dependencies/dependency-apps-table.tsx

```diff
@@ -229,3 +237,32 @@ export const DependencyAppsTable: React.FC<IDependencyAppsTableProps> = ({
     </>
   );
 };
+
+const DependencyVersionColumn = ({
+  appDependency: {
+    dependency: {
+      provider,
+      name,
+      version,
+      rule: sha, // TODO: rename to 'sha' with https://github.com/konveyor/tackle2-hub/issues/557
```

Comment:
LGTM - wondering if we should hold off for the hub PR to merge?

Reply:
Option 1 - hold off and update this PR at the same time

Option 2 - merge this PR and fix the payload with #1536 

I'm leaning towards option 2, but it is not a strong preference.

Reply:
Merging and circling back sounds ok to me.